docs/installing-lagoon/update-lagoon.md

```diff
@@ -1,15 +1,53 @@
 # Updating
 
-1. Follow normal Helm Chart update procedures.
-2. Tell Helm to download newest charts:
-
-    `helm repo update`
-3. Check with `helm diff` for changes: (https://github.com/databus23/helm-diff if not installed).
-   1. `helm diff upgrade --install --create-namespace --namespace lagoon-core -f values.yaml lagoon-core lagoon/lagoon-core`
-4. Upgrade:
-   1. `helm upgrade --install --create-namespace --namespace lagoon-core -f values.yaml lagoon-core lagoon/lagoon-core`
-5. If upgrading Lagoon Core:
-   1. `kubectl exec -it lagoon-core-api-db-0 sh`
-   2. Run inside shell:
-      1. `/rerun_initdb.sh`
-6. Check [https://github.com/uselagoon/lagoon/releases](https://github.com/uselagoon/lagoon/releases) for additional upgrades.
+1. Download newest charts using helm
+
+    ```
+    helm repo update
+    ```
+
+2. Check with `helm diff` for changes [https://github.com/databus23/helm-diff](https://github.com/databus23/helm-diff).
+
+    ```
+    helm diff upgrade --install --create-namespace --namespace lagoon-core \
+        -f values.yaml lagoon-core lagoon/lagoon-core
+    ```
+
+3. Run the upgrade using helm
+
+    ```
+    helm upgrade --install --create-namespace --namespace lagoon-core \
+        -f values.yaml lagoon-core lagoon/lagoon-core
+    ```
+
+4. If upgrading Lagoon Core, ensure you run the `rerun_initdb.sh` script to perform post upgrade migrations
+
+    ```
+    kubectl --namespace lagoon-core exec -it lagoon-core-api-db-0 -- \
+        sh -c /rerun_initdb.sh
+    ```
+
```

Comment:
Might also be worth including the instructions to sync groups to OpenSearch
```suggestion

5. If upgrading Lagoon Core, and you have enabled groups/user syncing for OpenSearch, you may additionally need to run the `sync:opendistro-security` script to update the groups in OpenSearch. This command can also be prefixed with a `GROUP_REGEX=<group-to-sync` to sync a single group at a time, as syncing the entire group structure may take a long time. 

    ```
    kubectl --namespace lagoon-core exec -it deploy/lagoon-core-api -- \
        sh -c yarn sync:opendistro-security
    ```
```